static/js/pages/instances/InstanceDetail.tsx

```diff
@@ -21,6 +29,27 @@ const InstanceDetail: FC = () => {
     return <>Missing name</>;
   }
 
+  const {
+    data: instance,
+    error,
+    isLoading,
+  } = useQuery({
+    queryKey: [queryKeys.instances, name],
+    queryFn: () => fetchInstance(name),
+  });
+
+  if (error) {
+    notify.failure("Could not load instance details.", error);
+  }
+
+  if (isLoading) {
+    return <Loader text="Loading instance details..." />;
```

Comment:
I think the loading and error state when no instance is present should be displayed within the BaseLayout. It could both be in a row instead of the tabs. Otherwise we render a just white page with the loader/error.

Reply:
Done @ [4f62c9e](https://github.com/canonical/lxd-ui/commit/4f62c9e2df86d2b692c4298deac8212f25ffcc02)